Framework/Foundation/include/Framework/Pack.h

```diff
@@ -272,6 +272,28 @@ constexpr auto unique_pack(pack<T, Ts...>, PT p2)
 template <typename P>
 using unique_pack_t = decltype(unique_pack(P{}, pack<>{}));
 
+template <typename... Ts>
+inline constexpr std::tuple<Ts...> pack_to_tuple(pack<Ts...>)
+{
+  return std::tuple<Ts...>{};
+}
+
+template <typename P>
+using pack_to_tuple_t = decltype(pack_to_tuple(P{}));
+
+template <typename T, std::size_t N, typename... REST>
```

Comment:
Why do you need REST at all? Isn't T the  only type involved?

Reply:
Ok, nevermind, I understand the logic now.

Reply:
To repeat T. REST is nothing, then T, then T, T and so on, till REST becomes T repeated N times.